tags/resources/toc-supporting-guides/general-technical-questions.md

```diff
@@ -0,0 +1,189 @@
+  * Explain the primary use case for the project. What additional use cases are supported by the project?
+  * Explain which use cases have been identified as unsupported by the project.  
+  * Describe the intended types of organizations who would benefit from adopting this project. (i.e. financial services, any software manufacturer, organizations providing platform engineering services)?  
+  * Please describe any completed end user research and link to any reports.
+
+### Usability
+
+  * How should the target personas interact with your project?  
+  * Describe the user experience (UX) and user interface (UI) of the project.  
+  * Describe how this project integrates with other projects in a production environment.
+
+### Design
+
+  * Explain the design principles and best practices the project is following.   
+  * Outline or link to the project’s architecture requirements? Describe how they differ for Proof of Concept, Development, Test and Production environments, as applicable.  
+  * Define any specific service dependencies the project relies on in the cluster.  
+  * Describe how the project implements Identity and Access Management.  
+  * Describe how the project has addressed sovereignty.  
+  * Describe any compliance requirements addressed by the project.  
+  * Describe the project’s High Availability requirements.  
+  * Describe the project’s resource requirements, including CPU, Network and Memory.  
+  * Describe the project’s storage requirements, including its use of ephemeral and/or persistent storage.  
+  * Please outline the project’s API Design:  
+    * Describe the project’s API topology and conventions  
+    * Describe the project defaults  
+    * Outline any additional configurations from default to make reasonable use of the project  
+    * Describe any new or changed API types and calls \- including to cloud providers \- that will result from this project being enabled and used  
+    * Describe compatibility of any new or changed APIs with API servers, including the Kubernetes API server   
+    * Describe versioning of any new or changed APIs, including how breaking changes are handled  
+  * Describe the project’s release processes, including major, minor and patch releases.
+
+### Installation
+
+  * Describe how the project is installed and initialized, e.g. a minimal install with a few lines of code or does it require more complex integration and configuration?  
+  * How does an adopter test and validate the installation?
+
+### Security
+
+  * Projects   
+  * Please provide a link to the project’s cloud native [security self assessment](https://tag-security.cncf.io/community/assessments/).  
+  * Please review the [Cloud Native Security Tenets](https://github.com/cncf/tag-security/blob/main/security-whitepaper/secure-defaults-cloud-native-8.md) from TAG Security.  
+    * How are you satisfying the tenets of cloud native security projects?  
```

Comment:
Is there a link for this?

Reply:
the Tenets link is in the bullet above

Reply:
Sorry, didn't realize it was the same link